script/tool/lib/src/license_check_command.dart

```diff
@@ -63,8 +63,6 @@ final RegExp _workivaLicenseRegex = RegExp(
 // TODO(stuartmorgan): Replace this with a single string once all the copyrights
 // are standardized.
 final List<String> _firstPartyAuthors = <String>[
-  'The Chromium Authors',
-  'the Chromium project authors',
```

Comment:
is it possible to simplify this now?

Reply:
Yes, I was fixing the 'the Flutter project authors' in parallel in another PR and forgot to simplify here when I rebased. I'll change this to a string rather than a list; more simplification will come with the next mega-PR, which will do dates.